streams/src/main/java/org/apache/kafka/streams/processor/internals/StreamThread.java

```diff
@@ -742,9 +742,9 @@ boolean runLoop() {
                     errorMessage.startsWith("Broker unexpectedly doesn't support requireStable flag on version ")) {
 
                     log.error("Shutting down because the Kafka cluster seems to be on a too old version. " +
-                              "Setting {}=\"{}\"/\"{}\" requires broker version 2.5 or higher.",
+                              "Setting {}=\"{}\" requires broker version 2.5 or higher.",
                           StreamsConfig.PROCESSING_GUARANTEE_CONFIG,
-                          StreamsConfig.EXACTLY_ONCE_V2, StreamsConfig.EXACTLY_ONCE_BETA);
```

Comment:
Could you please remove the `@SuppressWarnings("deprecation")` from line:688 as `StreamsConfig.EXACTLY_ONCE_BETA` is removed

Reply:
Good point. Will do in a follow up PR. (EOSv1 removal will be many PRs...)

Reply:
Addressed via https://github.com/apache/kafka/pull/17170